packages/sources/dns-query/src/index.ts

```diff
@@ -1,7 +1,8 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
-import { makeConfig } from './config'
+import { makeConfig, NAME } from './config'
+import rateLimit from './config/limits.json'
 
-const NAME = 'DNS_QUERY'
+const adapterContext = { name: NAME, rateLimit }
 
```

Comment:
Since we are changing this line, could we modify this to be standard with a standard export style?
```suggestion
const { server } = expose(adapterContext, makeExecute(), undefined, endpointSelector)
```